### ISSUE TITLE
Remove dependencies

### ISSUE THREAD

**Issue by binyamin** [bukalapak/pkce-npm#1]
According to [this article](https://www.loginradius.com/engineering/blog/pkce/), nodejs's native crypto module can do what crypto-js and secure-random are accomplishing here.

**Comment by binyamin** [comment]
See https://github.com/binyamin/pkce-gen

Do you want me to make a PR?

**Comment by binyamin** [comment]
@subosito friendly ping

**Comment by subosito** [comment]
Hi @binyamin, this package is intended to work in the browser, I don't think we can use the native nodejs module for that. I tried using browserify previously, but not working. Any idea?

**Comment by binyamin** [comment]
Ah, gotcha. Well then, have a look at https://github.com/aaronpk/pkce-vanilla-js

**Comment by subosito** [comment]
I don't think that one works on Node.js. It looks like I forgot to put "(also) in the browser" in previous comment :D.

**Comment by binyamin** [comment]
I don't really understand how this would work in the browser, since it uses "require". Unless you're using a bundler, you can't "require" external modules with pure, vanilla JS.

Another thought: maybe write a browser version and a node version, and then modify the package.json so a cdn would serve the the correct file.
